MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -279,6 +410,18 @@ public void SwitchCommandNonFatal (WebServiceLogin login, int command_id)
 		}
 
 		[WebMethod]
+		public void SwitchCommandAlwaysExecute (WebServiceLogin login, int lane_id, int command_id)
+		{
+			using (DB db = new DB ()) {
+				var lane = DBLane_Extensions.Create (db, lane_id);
```

Comment:
You're not verifying that `command_id`'s lane is `lane_id`, which means someone can call this method for all command_ids as long as they have permission to change one lane.